addons/mrp/views/product_views.xml

```diff
@@ -41,7 +41,7 @@
         <record id="product_template_form_view_bom_button" model="ir.ui.view">
             <field name="name">product.template.procurement</field>
             <field name="model">product.template</field>
-            <field name="inherit_id" ref="product.product_template_only_form_view"/>
+            <field name="inherit_id" ref="product.product_template_form_view"/>
             <field name="groups_id" eval="[(4, ref('mrp.group_mrp_user'))]"/>
             <field name="arch" type="xml">
                 <button name="toggle_active" position="before">
```

Comment:
not working for product. check here for template it's open action and in variant it's define function which call different action with domain..